hwilib/devices/coldcard.py

```diff
@@ -106,3 +115,34 @@ def wipe_device(self):
     # Close the device
     def close(self):
         self.device.close()
+
+def enumerate(password=None):
+    results = []
+    for d in hid.enumerate(COINKITE_VID, CKCC_PID):
+        d_data = {}
+
+        path = d['path'].decode()
+        d_data['type'] = 'coldcard'
+        d_data['path'] = path
+
+        try:
+            client = ColdcardClient(path)
+            master_xpub = client.get_pubkey_at_path('m/0h')['xpub']
+            d_data['fingerprint'] = get_xpub_fingerprint_hex(master_xpub)
+            client.close()
+        except Exception as e:
+            d_data['error'] = "Could not open client or get fingerprint information: " + str(e)
+
+        results.append(d_data)
+    # Check if the simulator is there
+    try:
+        client = ColdcardClient(CC_SIMULATOR_SOCK)
```

Comment:
Can a tutorial on setting up the simulator be linked?

Reply:
Some documentation for it is here: https://github.com/Coldcard/firmware/blob/master/unix/README.md

Although I haven't quite figured it out for myself either.

I added a comment with a link to the docs.